### ISSUE TITLE
Dependency of multiparty ~ 3.3.1 causes deprecation warning from os.tmpDir

### ISSUE THREAD

**Issue by CaptainJimmy** [SpiderStrategies/node-gmail-api#37]
[DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.

traced that to:

~/environment/gmailSeeder/node_modules $ grep -rnw . -e 'os.tmpDir'                                                                                                                                                                                          
./node-gmail-api/node_modules/multiparty/README.md:88:   Defaults to `os.tmpDir()`.
./node-gmail-api/node_modules/multiparty/index.js:55:  self.uploadDir = options.uploadDir || os.tmpDir();

**Comment by QaiserMajeed** [comment]
any update on this ? facing the error

**Comment by nathanbowser** [comment]
PR welcome.

**Comment by gustawdaniel-acaisoft** [comment]
@nathanbowser to solve this issue there is required publishing current master code to npm.

---

This problem is described here

https://github.com/alhazmy13/serverless-offline-python/issues/18

I can reproduce these scenarios:
1. node 12 - works
2. node 14 and 16 error

/home/daniel/exp/google-api-test/node_modules/multiparty/index.js:55
  self.uploadDir = options.uploadDir || os.tmpDir();
                                           ^

TypeError: os.tmpDir is not a function
    at new Form (/home/daniel/exp/google-api-test/node_modules/multiparty/index.js:55:44)
    at Request.<anonymous> (/home/daniel/exp/google-api-test/node_modules/node-gmail-api/index.js:117:20)
    at Request.emit (events.js:376:20)
    at Request.onRequestResponse (/home/daniel/exp/google-api-test/node_modules/request/request.js:1059:10)
    at ClientRequest.emit (events.js:376:20)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:647:27)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)
    at TLSSocket.socketOnData (_http_client.js:515:22)
    at TLSSocket.emit (events.js:376:20)
    at addChunk (internal/streams/readable.js:309:12)

problem is in `request` package that is depreciated.

https://betterprogramming.pub/request-has-been-deprecated-a76415b4910b

there are listed replacements of `request`

https://github.com/request/request/issues/3143

Line 

```
os.tmpDir = os.tmpdir
```

in index of `multiparty` solve problem.

PR to multiparty created

https://github.com/pillarjs/multiparty/pull/242

---

This problem was solved in multiparty version 4.1.3 / 2017-01-22

https://github.com/pillarjs/multiparty/blob/master/HISTORY.md#413--2017-01-22

but in my project old multiparty version is installed

```
cat package-lock.json| grep multiparty
    "node_modules/multiparty": {
      "resolved": "https://registry.npmjs.org/multiparty/-/multiparty-3.3.2.tgz",
        "multiparty": "^3.3.1",
    "multiparty": {
      "resolved": "https://registry.npmjs.org/multiparty/-/multiparty-3.3.2.tgz",
        "multiparty": "^3.3.1",
```

it is becuse of npm version of `node-gmail-api` 1.0.0 was published on npm a year ago with deplendencies

```
  "dependencies": {
    "multiparty": "^3.3.1",
    "request": "^2.40.0",
    "split": "^0.3.0",
    "stream-stream": "^1.2.6",
    "through2": "^2.0.0"
  },
```

https://www.npmjs.com/package/node-gmail-api

but current master has updated deplendencies

```
  "dependencies": {
    "multiparty": "^4.2.2",
    "request": "^2.40.0",
    "split": "^0.3.0",
    "stream-stream": "^1.2.6",
    "through2": "^2.0.0"
  },
  ```
  
  but is not published on `npm`.

**Comment by gustawdaniel-acaisoft** [comment]
@nathanbowser this is important issue for me.

For other users with this problem current sollution is:

```
 npm install https://github.com/SpiderStrategies/node-gmail-api
```

then multiparty has proper version

```
cat package-lock.json| grep multiparty
    "node_modules/multiparty": {
      "resolved": "https://registry.npmjs.org/multiparty/-/multiparty-4.2.2.tgz",
    "node_modules/multiparty/node_modules/http-errors": {
    "node_modules/multiparty/node_modules/safe-buffer": {
    "node_modules/multiparty/node_modules/setprototypeof": {
        "multiparty": "^4.2.2",
    "multiparty": {
      "resolved": "https://registry.npmjs.org/multiparty/-/multiparty-4.2.2.tgz",
        "multiparty": "^4.2.2",
```

**Comment by nathanbowser** [comment]
Just published v1.0.1
